src/components/SearchView/DeletableRow.tsx

```diff
@@ -0,0 +1,87 @@
+import React, { Component } from 'react';
```

Comment:
A fair amount of this code was copied (and then modified) from:
https://github.com/software-mansion/react-native-gesture-handler/blob/master/examples/Example/swipeable/AppleStyleSwipeableRow.js

TODO: since react-native-gesture-handler is already included in package.json, do I need to do any special attribution here or elsewhere?